index.js

```diff
@@ -772,12 +772,12 @@ module.exports = {
         return list
     },
 
-    getFileMetaDataFromPath: async function (allocation_id, path, client_id) {
+    getFileMetaDataFromPath: async function (allocation_id, path, client_id,public_key) {
```

Comment:
I didn't notice this before, but it'd be nice to keep good formatting.
Same applies to all other lines:
```suggestion
    getFileMetaDataFromPath: async function (allocation_id, path, client_id, public_key) {
```